registry.tcl

```diff
@@ -783,4 +783,9 @@ extension EGL_QNX_image_native_buffer {
     flags       public
     filename    extensions/QNX/EGL_QNX_image_native_buffer.txt
 }
-# Next free extension number: 152
+extension EGL_EXT_query_context {
```

Comment:
extension name is not a match

Reply:
I renamed the extension halfway through. Thanks for catching this.